cmd/minishift/cmd/addon/addon.go

```diff
@@ -28,9 +28,10 @@ const (
 )
 
 var AddonsCmd = &cobra.Command{
-	Use:   "addons SUBCOMMAND [flags]",
-	Short: "Manages Minishift add-ons",
-	Long:  "Manages Minishift add-ons. You can install, list, enable or disable Minishift add-ons.",
+	Use:     "addons SUBCOMMAND [flags]",
+	Aliases: []string{"addon"},
```

Comment:
> resources are supposed to be plural for an API. I believe I made the hostfolders singular... but not happy with this. I'd rather make plural the default case and add singular as alias.

A bit confused here, since I feel your answer is partly contradicitve

> resources are supposed to be plural for an API

Not sure about this one. OOI why do you think so. Also API is imo different from a CLI where I find plural quite uncommon, especially in the Unix works where you even shave of vocals. But anyways, ...

> I believe I made the hostfolders singular... but not happy with this

Right, so _hostfolder_ is singular. Personally I like it and I seem to recall we even having this discussion and at the time decided for singular.

>  but not happy with this.

I am :-)

> I'd rather make plural the default case and add singular as alias.

Here I get confused. Are you referring to _hostfolder_ here? Since for add-ons we have exactly this case now. Note, my driving force for 'addons' as default was purely the alignment with `minikube`. Personally I would have chosen singular.

That all said, the beauty of Viper is, that we don't have to care so much. We can easily have both. The question might be around which default to choose, but in case of 'addons' this decision is basically made. Whether we want to align 'hostfolder' is a different question. Open for making 'host folders' the default and alias to 'hostfolder'. However, I don't think I have to do anything in the context of this pull request right now.